inference-engine/src/inference_engine/src/cpp_interfaces/interface/ie_iexecutable_network_internal.cpp

```diff
@@ -26,6 +26,10 @@ void IExecutableNetworkInternal::setNetworkOutputs(const OutputsDataMap& network
     _networkOutputs = networkOutputs;
 }
 
+void IExecutableNetworkInternal::setFunction(const std::shared_ptr<ov::Function>& function) {
```

Comment:
`setRuntimeFunction`? according to https://github.com/openvinotoolkit/openvino/pull/7230/files

Reply:
done